http/relay/relay_test.go

```diff
@@ -21,14 +24,55 @@ func TestRelay(t *testing.T) {
 	c := &up.Config{}
 	assert.NoError(t, c.Default(), "default")
 
-	h, err := New(c)
-	assert.NoError(t, err, "init")
+	var h http.Handler
+	newH := func(t *testing.T) {
+		localH, err := New(c)
+		assert.NoError(t, err, "init")
+		h = localH
+	}
 
-	t.Run("GET simple", func(t *testing.T) {
+	// newRequestBody is a helper to fetch the body from a child process route
+	newRequestBody := func(t *testing.T, method, path string, body io.Reader) string {
+		t.Helper()
```

Comment:
Not sure if you're aware, but if you use this and have failing tests - `t.Helper()` tells go to skip this stack frame for error reporting line numbers and use the parent stack frame.  VERY helpful for knowing which tests actually failed.

Reply:
nice! yea I had read about it in the release notes but I haven't tried it